application/view/omeka/admin/media/show-details.phtml

```diff
@@ -25,6 +25,13 @@ $item = $resource->item();
     </div>
     <?php endif; ?>
 
+    <?php if (is_numeric($resource->size())): ?>
```

Comment:
The value must either be a number or null, right? Seems these should be "not null" checks... is_numeric more appropriate when the value is a number-ish string.

Reply:
Doctrine returns string for bigint data types.

Reply:
ok then